src/server/rdb_save.cc

```diff
@@ -1644,6 +1638,8 @@ void SerializerBase::CompressBlob() {
   memcpy(dest.data(), compressed_blob.data(), compressed_blob.length());
   mem_buf_.CommitWrite(compressed_blob.length());
   ++compression_stats_->compressed_blobs;
+  auto& stats = ServerState::tlocal()->stats;
```

Comment:
nit: ugh, we already have `compression_stats_` that go nowhere and now we also have this. Shoud we, maybe, expose `compression_stats_` via API and then record the stats in the upper layer?

Reply:
are you suggesting something like `debug compression` ?
I can do that, but I suggest to push it in a new PR.
If you dont have more comments on this one we can merge it and you can pull this to your change with lz4 to test and I can work at the same time on this new api